mozdownload/scraper.py

```diff
@@ -639,7 +639,9 @@ def get_build_info(self):
             self.build_index = 0
             self.logger.info('Selected build: build%s' % self.build_number)
         else:
-            self.logger.info('Selected build: build%s' % self.build_index)
+            # Otherwise displays incorrect build number
+            display_index = str(int(self.build_index) + 1)
+            self.logger.info('Selected build: build%s' % display_index)
```

Comment:
Isn't build_index always a number? Also the formatting can automatically convert it to a string. Just pass in the `(index + 1)` in brackets.